### ISSUE TITLE
SWS

### ISSUE THREAD

**Issue by qianweihao** [kthorp/pyfao56#43]
Hello, I have a problem, if I provide Zrmax, but due to the influence of the measuring instrument, the sws data I measured does not reach Zrmax, this will cause the IndexError problem, is there a way to solve this problem

**Comment by qianweihao** [comment]
Thank you. If my measured value does not reach Zrmax, I can assume that Zrmax is the deepest value I measured. Is that what it means?

**Comment by kthorp** [comment]
Yes, or alternatively, you can specify the depth of the deepest measurement layer as Zrmax.  Then it would assume that the deepest measurement layer extends downward to the depth of Zrmax.

**Comment by qianweihao** [comment]
I see. Thanks for the answer.
